c2corg_ui/static/js/addassociation.js

```diff
@@ -26,9 +26,8 @@ app.addAssociationDirective = function($compile) {
       'dataset' : '='
     },
     controllerAs: 'addCtrl',
-
     link: function(scope, element, attrs, ctrl) {
-      element.html(template(ctrl.dataset));
+      element.html(template(ctrl.dataset_));
```

Comment:
Just curious: if you access a property of the controller from within the directive itself, is this property really private? Shouldn't it be ``@export``ed?